src/test/java/de/tum/in/www1/artemis/exercise/programmingexercise/ProgrammingExerciseUtilService.java

```diff
@@ -176,89 +223,190 @@ public ProgrammingSubmission createProgrammingSubmission(Participation participa
         return submissionRepository.save(programmingSubmission);
     }
 
+    /**
+     * Creates and saves an already submitted programming submission done manually. "9b3a9bd71a0d80e5bbc42204c319ed3d1d4f0d6d" is used as the commit hash.
```

Comment:
```suggestion
     * Creates and saves an already submitted programming submission done manually. <code>9b3a9bd71a0d80e5bbc42204c319ed3d1d4f0d6d</code> is used as the commit hash.
```
I suggest this usage for all similar cases. Looks better in the rendered doc, imo

Reply:
Done, it does look better :+1: